README.md

```diff
@@ -62,7 +62,7 @@ environment with the following packages:
  * shapely
  * cartopy >= 0.18.0
  * cartopy\_offlinedata
- * geometric\_features
+ * geometric\_features >= 0.1.9
```

Comment:
It isn't actually strictly necessary that we have the latest `geometric_features` at the moment unless you want to re-create the preprocessed `geojson` files, but it certainly doesn't hurt.